spec.bs

```diff
@@ -40,6 +40,9 @@ spec: webdriver; urlPrefix: https://w3c.github.io/webdriver/
     type: dfn
         text: getting a property; url: dfn-getting-properties
         text: error code; url: dfn-error-code
+spec: fenced-frame; urlPrefix: https://wicg.github.io/fenced-frame/
+    type: dfn;
+        for: browsing context; text: fenced frame config instance
```

Comment:
This is not actually linking to the right thing automatically. We might need to add `url: #browsing-context-fenced-frame-config-instance`

Reply:
Added the url to the entry.